features/step_definitions/wallets-steps.js

```diff
@@ -50,9 +50,14 @@ Given(/^I have the following wallets:$/, async function (table) {
   await createWallets(table.hashes(), this);
 });
 
+// Does not automatically trigger the Data Layer Migration
+Given(/^I have the following wallets, skiping the data layer migration:$/, async function (table) {
```

Comment:
@daniloprates as described in my earlier comment if we move the migration layer acceptance in the basic setup background step we don't even need this specific step definition.